src/__tests__/__snapshots__/profiling-test.js.snap

```diff
@@ -55,63 +55,48 @@ Object {
 }
 `;
 
-exports[`profiling CommitDetails should be collected for each commit: exported data 1`] = `
+exports[`profiling CommitDetails should be collected for each commit: imported data 1`] = `
 Object {
   "commitDetails": Array [
     Object {
-      "actualDurations": Array [
-        1,
-        12,
-        2,
-        12,
-        3,
-        0,
-        4,
-        1,
-        5,
-        1,
-      ],
+      "actualDurations": Map {
+        1 => 12,
+        2 => 12,
+        3 => 0,
+        4 => 1,
+        5 => 1,
+      },
```

Comment:
Note this snapshot change, this was the bug that I was talking about, mismatch Array vs Map.